### ISSUE TITLE
Logical keyboard navigation for global layout

### ISSUE THREAD

**Issue by seanes** [Altinn/altinn-components#560]
Components in global layout, i.e. header (logo, search bar, language selector, menu), sidebar and content should be placed logically in markdown (or in worst case act as of they are) so that they follow correct order in keyboard navigation (i.e. tabs). 

There is a slight change in layout for mobile/tablets and desktop and this should be taken into account.
